client/src/components/profile/components/internet.tsx

```diff
@@ -163,136 +138,130 @@ class InternetSettings extends Component<InternetProps, InternetState> {
-                )}
-                <Info message={linkedinValidationMessage} />
-              </FormGroup>
-              <FormGroup
-                controlId='internet-twitter'
-                validationState={twitterValidation}
-              >
-                <ControlLabel htmlFor='internet-twitter-input'>
-                  Twitter
-                </ControlLabel>
-                <FormControl
-                  onChange={this.createHandleChange('twitter')}
-                  placeholder='https://twitter.com/user-name'
-                  type='url'
-                  value={twitter}
-                  id='internet-twitter-input'
-                />
-                {this.renderCheck(
-                  twitter,
-                  twitterValidation,
-                  'internet-twitter-check'
-                )}
-                <Info message={twitterValidationMessage} />
-              </FormGroup>
-              <FormGroup
-                controlId='internet-website'
-                validationState={websiteValidation}
-              >
-                <ControlLabel htmlFor='internet-website-input'>
-                  {t('settings.labels.personal')}
-                </ControlLabel>
-                <FormControl
-                  onChange={this.createHandleChange('website')}
-                  placeholder='https://example.com'
-                  type='url'
-                  value={website}
-                  id='internet-website-input'
-                />
-                {this.renderCheck(
-                  website,
-                  websiteValidation,
-                  'internet-website-check'
-                )}
-                <Info message={websiteValidationMessage} />
-              </FormGroup>
-            </div>
-            <BlockSaveButton
-              disabled={isDisabled}
-              bgSize='large'
-              {...(isDisabled && { tabIndex: -1 })}
+  return (
+    <>
+      <SectionHeader>{t('settings.headings.internet')}</SectionHeader>
+      <FullWidthRow>
+        <form
+          id='internet-presence'
+          onSubmit={handleSubmit}
+          data-playwright-test-label='internet-presence'
+        >
+          <div role='group' aria-label={t('settings.headings.internet')}>
+            <FormGroup
+              controlId='internet-github'
+              validationState={githubProfileValidation}
             >
-              {t('buttons.save')}{' '}
-              <span className='sr-only'>{t('settings.headings.internet')}</span>
-            </BlockSaveButton>
-          </form>
-        </FullWidthRow>
-      </>
-    );
-  }
-}
+              <ControlLabel htmlFor='internet-github-input'>
+                GitHub
+              </ControlLabel>
+              <FormControl
+                data-playwright-test-label='internet-github-input'
+                onChange={createHandleChange('githubProfile')}
+                placeholder='https://github.com/your-handle'
```

Comment:
I don't have strong feelings either way, but this is changing behaviour. If you want to do that, do you mind creating a separate PR for these?

Reply:
I'll change it back, it doesn't do much difference either way.